expression/builtin_math_vec.go

```diff
@@ -807,11 +807,25 @@ func (b *builtinFloorIntToDecSig) vecEvalDecimal(input *chunk.Chunk, result *chu
 }
 
 func (b *builtinSignSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinSignSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	if err := b.args[0].VecEvalReal(b.ctx, input, result); err != nil {
+		return err
+	}
+	i64s := result.Int64s()
```

Comment:
We can't call `result.Int64s` since `result` is used to store real datums in `args[0].VecEvalReal`.

Reply:
oh!  i understand !  
I need to use `bufAllocator`  to store intermediate results.